File: java/src/main/java/bouchonnois/domain/Event.java
package bouchonnois.domain;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public record Event(LocalDateTime date, String message) {
    @Override
    public String toString() {
        return date.format(DateTimeFormatter.ofPattern("HH:mm - ")) + message;
    }
}

File: kotlin/src/main/kotlin/bouchonnois/domain/Event.kt
package bouchonnois.domain

import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

data class Event(val date: LocalDateTime, val message: String) {
    override fun toString(): String {
        return date.format(DateTimeFormatter.ofPattern("HH:mm - ")) + message
    }
}